### PR TITLE
added linter props validations

### PR DESCRIPTION
**Pull request by frankctan**

### CODE REVIEW

File: src/ZulipApp.js [hunk 1]
```diff
@@ -33,16 +33,25 @@ const ZulipApp = (props) => {
...
 });
 
-const mapDispatchToProps = (dispatch, ownProps) =>
+const mapDispatchToProps = (dispatch) =>
   bindActionCreators({
     attemptLogin,
     getLatestMessages,
   }, dispatch);
 
+
```
Comment (neerajwahi) [inline src/ZulipApp.js]: Remove extra line

File: src/account/ZulipLoginView.js [hunk 2]
```diff
@@ -63,13 +63,18 @@ const ZulipLoginButton = (props) => (
...
+
 export default class ZulipLoginView extends Component {
   constructor(props) {
     super(props);
     this.state = {
       realm: 'http://localhost:9991',
-      email: 'cordelia@zulip.com',
-      password: 'testing123',
+      email: 'test@test.com',
+      password: 'password',
```
Comment (neerajwahi) [inline src/account/ZulipLoginView.js]: Could you remove this default dev user/password and replace with an empty string? This will likely vary between people's dev setups.

File: src/nav/ZulipNavBar.ios.js [hunk 3]
```diff
@@ -50,7 +50,7 @@ const styles = StyleSheet.create({
 
 const ZulipNavBar = (props) => (
   <Navigator
-    initialRoute={{ name: 'All Streams', index: 0 }}
+    initialRoute={{ name: 'Hello world!', index: 0 }}
```
Comment (neerajwahi) [inline src/nav/ZulipNavBar.ios.js]: Please undo this change

**Comment by neerajwahi** [comment]
Thanks @frankctan! Looks good overall. I wrote some inline comments.

Could you please rebase this onto master instead of merging? That way we can maintain a linear git history.

**Comment by neerajwahi** [comment]
Oh, one more thing, could you change the commit message to something like "Add propTypes to React components." Trying to conform to this commit message style: http://zulip.readthedocs.io/en/latest/code-style.html#commit-messages

**Comment by borisyankov** [comment]
The project is already configured to use Flow (as all new React Native projects are) and it replaces (and is superior to) the PropTypes.

This can be closed, I'll add props type annotations soon.
